lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -265,11 +313,16 @@ const Reportback = {
 
     for (let i = 0, count = items.length; i < count; i++) {
```

Comment:
not blocking this PR, more of a question. how ES6-y are we trying to be? eg: this could be

```
for (let item of items) {
  ...
}
```